hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/JetGetJobConfigCodec.java

```diff
@@ -35,37 +35,63 @@
 
 /**
  */
-@Generated("2d11d0f610472f47fdd231cf4ddd7d9b")
+@Generated("e126087ccfb795f19da96740a1d56942")
 public final class JetGetJobConfigCodec {
     //hex: 0xFE0800
     public static final int REQUEST_MESSAGE_TYPE = 16648192;
     //hex: 0xFE0801
     public static final int RESPONSE_MESSAGE_TYPE = 16648193;
     private static final int REQUEST_JOB_ID_FIELD_OFFSET = PARTITION_ID_FIELD_OFFSET + INT_SIZE_IN_BYTES;
-    private static final int REQUEST_INITIAL_FRAME_SIZE = REQUEST_JOB_ID_FIELD_OFFSET + LONG_SIZE_IN_BYTES;
+    private static final int REQUEST_LIGHT_JOB_COORDINATOR_FIELD_OFFSET = REQUEST_JOB_ID_FIELD_OFFSET + LONG_SIZE_IN_BYTES;
+    private static final int REQUEST_INITIAL_FRAME_SIZE = REQUEST_LIGHT_JOB_COORDINATOR_FIELD_OFFSET + UUID_SIZE_IN_BYTES;
     private static final int RESPONSE_INITIAL_FRAME_SIZE = RESPONSE_BACKUP_ACKS_FIELD_OFFSET + BYTE_SIZE_IN_BYTES;
 
     private JetGetJobConfigCodec() {
     }
 
-    public static ClientMessage encodeRequest(long jobId) {
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
+    public static class RequestParameters {
+
+        /**
+         */
+        public long jobId;
+
+        /**
+         */
+        public @Nullable java.util.UUID lightJobCoordinator;
```

Comment:
If I am not mistaken, we are using this just to see if the job is a light job or not, right?

If so, are we doing this because we might need this information in the future?

Reply:
Yes, we need information if the job is a light job. We pass that information to ```GetJobConfigOperation``` and then to ```com.hazelcast.jet.impl.JetServiceBackend#getJobConfig```

Reply:
I wanted to be consistent with other part of Hazelcast code. ```JetGetJobSubmissionTimeMessageTask``` has exactly same structure.

Reply:
Oh okay. I missed the part that `getLightJobCoordinator` is overridden from the `AbstractJetMessageTask`. It's all good then :+1: